transmit-lib/src/cmr/transmit/association.clj

```diff
@@ -28,6 +28,19 @@
   [context concept-id]
   (associations-by-concept-ids-url context :variable concept-id))
 
+(defmulti single-association-url
```

Comment:
I would do the generalization when we have a use case later, but you can keep it since you already did it.